catalog/format/iceberg/src/test/java/org/projectnessie/catalog/formats/iceberg/types/TestIcebergTypes.java

```diff
@@ -125,10 +125,22 @@ static Stream<Arguments> icebergTypes() {
         arguments(Types.DateType.get(), dateType()),
         arguments(Types.TimeType.get(), timeType()),
         arguments(
-            Types.StructType.of(Types.NestedField.of(11, false, "field11", Types.StringType.get())),
+            Types.StructType.of(
+                Types.NestedField.builder()
```

Comment:
Contrary to what the javadocs of `of()` say, the best replacement is not the builder, but the `optional()` or `required()` methods.

Reply:
I :heart: javadocs ...